docs/aerodromes/classd/Bankstown.md

```diff
@@ -139,4 +161,14 @@ VFR aircraft require a level readback.
     <span class="hotline">**BK ADC** -> **SY TCU**</span>: "UJN, A010"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. You should *generally* be able to issue a clearance for an approach and use other separation methods (visual separation, holding a departure on the ground) if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. You should *generally* be able to issue a clearance for an approach and use other separation methods (visual separation, holding a departure on the ground) if separation is required with these aircraft.
+
+#### ADC 2 Online
+When ADC 2 is online, SY TCU may not be familiar with which controller owns what airspace. Best practice is to receive the coordination no matter what, and if it was meant for the other ADC controller, relay the coordination to them internally.
```

Comment:
I might just leave this in there, since ENR will probably coord to ADC 1 by default most of the time, just so the tower controllers know to distribute the information internally where required